update_ip/services/__init__.py

```diff
@@ -1,5 +1,37 @@
-from nfsn import NearlyFreeSpeechService
-from webfaction import WebFactionService
+import inspect
+import os
+import imp
+import glob
+from base import BaseDNSService
 
-services= [WebFactionService, NearlyFreeSpeechService]
```

Comment:
I think that hardcoded services are hard to maintain, even in this code are just 2 services and they don't are name sorted, if update_ip have 50 services tomorrow it will be a mess sorting manually all services.

And all services should be loaded since are created for using it, it's like a "plugins" folder, an option could be add a constant in services like "enabled=True" and check it before add to services list.